### ISSUE TITLE
Abstract logic satisfaction relation hardcoded

### ISSUE THREAD

**Issue by rzach** [OpenLogicProject/OpenLogic#55]
The Lindström's theorem chapter hard-codes the abstract logic satisfaction relation as \models_L.  Needs to be replaced by something defined in open-logic-config.sty à la \Sat

**Comment by rzach** [comment]
Right, @AldAnt ?  Maybe use a different symbol by default, say three-line turnstile?

**Comment by AldAnt** [comment]
We can use a different symbols, as long we are careful to preserve \models
for the classical relation: there will be instances of a logic <F,
\models_F> which is supposed to denote FOL, so such occurrences should not
be replaced.

A

On Sun, Sep 20, 2015 at 1:42 PM, Richard Zach <notifications@github.com>
wrote:

> Right, @AldAnt <https://github.com/AldAnt> ? Maybe use a different symbol
> by default, say three-line turnstile?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenLogicProject/OpenLogic/issues/55#issuecomment-141832760>
> .
>



-- 

*****************************************
G. Aldo Antonelli
Professor of Philosophy
University of California, Davis
http://aldo-antonelli.org
antonelli@ucdavis.edu +1 530 554 1368
